taxjar/taxjar-magento2-extension
# Add a sandbox toggle to the admin configuration

### Context



### Description

This change adds a toggle to the Tax admin configuration page, nested under debug mode.  When enabled, all API requests will be made to TaxJar's sandbox.  It also cleans up a small bug in Smartcalcs to standardize the way API calls are made.

![Screen Shot 2020-06-18 at 6 44 49 PM](https://user-images.githubusercontent.com/44789510/85085051-d6c89580-b193-11ea-9d33-6abc4d9cab49.png)

### Performance

Sandbox mode should only be used for testing.  This change has a negligible impact on production environments.  

### Testing

1.  Install the TaxJar extension,  connect the account to TaxJar and follow all setup instructions
2.  Enable debug mode and then enable sandbox mode
3.  Ensure all API requests are made to the sandbox endpoint (api.taxjar.sandbox.com)

#### Versions
